src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/SubmissionResultViewModel.java

```diff
@@ -26,7 +26,8 @@ public String getPresentableName() {
 
   @NotNull
   public String getStatusText() {
-    return getModel().getPoints() + "/" + getModel().getExercise().getMaxPoints() + " points";
+    String status = getModel().getPoints() + "/" + getModel().getExercise().getMaxPoints() + " points";
+    return (getModel().getStatus() == SubmissionResult.Status.UNKNOWN) ? "In grading" : status;
```

Comment:
Looks otherwise good but UI string "In grading" could be moved into resources (see e.g. 99a328d but here you can use simpler `getText` function instead of `getAndReplaceText`).

Reply:
Also a minor nit (that you don't have to address): now a value for `status` is computed even though it was never needed. Inlning `status` variable would avoid unnecessary computation.

Also, the result of `getModel()` call could be stored in a variable. Now it's called three times in a single method.

Reply:
Please make sure the code lines are no longer than 100 characters. The build fails because of this style error:
```
[ant:checkstyle] [WARN] /home/travis/build/Aalto-LeTech/intellij-plugin/src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/SubmissionResultViewModel.java:29: Line is longer than 100 characters (found 103). [LineLength]
```

You can run `gradlew check` on your local computer to make sure everything is right. (It may happen that 1-2 unit tests dealing with virtual file system fail in local, you don't have to care about those. Also, we have an issue #213 which may cause ~30 unit tests fail depending on your dev platform.)

Reply:
Yes, sure! I implemented your comments into the code:
- moved UI string "In grading" moved to `resources.properties`
- `status` variable computation moved to separate method
- `getModel()` stored in a variable

 I'll also produce a unit test. Closing the PR for the first time was my bad, sorry for unnecessary mess. I refactored the code to deal with too-long-lines error but some tests failed. I'm not sure if that's my bad or it's the probem with my dev platform.